allennlp/modules/seq2vec_encoders/pytorch_seq2vec_wrapper.py

```diff
@@ -19,11 +21,27 @@ class WrappedPytorchRnn(Seq2VecEncoder):
     RNNs.
     """
     def __init__(self, module: torch.nn.modules.RNNBase) -> None:
-        super(WrappedPytorchRnn, self).__init__()
+        super(PytorchSeq2VecWrapper, self).__init__()
         self._module = module
 
     def get_output_dim(self) -> int:
         return self._module.hidden_size * (2 if self._module.bidirectional else 1)
 
-    def forward(self, inputs: torch.Tensor) -> torch.Tensor:  # pylint: disable=arguments-differ
-        return self._module(inputs)[0][-1]
+    def forward(self,  # pylint: disable=arguments-differ
+                inputs: torch.Tensor,
+                sequence_lengths: torch.LongTensor = None) -> torch.Tensor:
+
+        if sequence_lengths is None:
+            return self._module(inputs)[0][:, :, -1]
+        sorted_inputs, sorted_sequence_lengths, restoration_indices = sort_batch_by_length(inputs,
+                                                                                           sequence_lengths)
+        packed_sequence_input = pack_padded_sequence(sorted_inputs, sorted_sequence_lengths, batch_first=True)
+
+        # Actually call the module on the sorted PackedSequence.
+        _, state = self._module(packed_sequence_input)
```

Comment:
I think longer term we want this to be configurable (e.g., do you concat the two directions, sum them, max pool them, max pool the state vectors for each timestep, etc.), but for now, I'd say just pick one that seems reasonable and make sure it's consistent in both cases.